docs/source/en/using-diffusers/text-img2vid.md

```diff
@@ -60,7 +50,6 @@ video = pipe(
     guidance_scale=6,
     generator=torch.Generator(device="cuda").manual_seed(42),
 ).frames[0]
-
 export_to_video(video, "output.mp4", fps=8)
 ```
 
```

Comment:
CogVideoX has a "video-to-video" variation, too. 
https://huggingface.co/docs/diffusers/main/en/api/pipelines/cogvideox#diffusers.CogVideoXVideoToVideoPipeline

Should this be mentioned?